wire-ios-transport/Source/Authentication/ZMPersistentCookieStorage.m

```diff
@@ -35,7 +35,7 @@
 static NSHTTPCookieAcceptPolicy cookiesPolicy = NSHTTPCookieAcceptPolicyAlways;
 
 
-static dispatch_queue_t isolationQueue(void)
+static dispatch_queue_t isolationQueue()
```

Comment:
@MarcinR Are you using Xcode 14.3? Are we sure that's going to compile using Xcode 14.2?